package.json

```diff
@@ -1,89 +1,129 @@
     }
   },
-  "engines": {
-    "node": ">=12.0.0"
+  "oclif": {
+    "commands": "./lib/commands",
+    "bin": "sfdx",
+    "devPlugins": [
+      "@oclif/plugin-help",
+      "@oclif/plugin-command-snapshot",
+      "@salesforce/plugin-command-reference"
+    ],
+    "topics": {
+      "force": {
+        "description": "tools for the Salesforce developer",
+        "subtopics": {
+          "data": {
+            "description": "manipulate records in your org",
+            "longDescription": "Use the data commands to manipulate records in your org. Commands are available to help you work with various APIs. Import CSV files with the Bulk API. Export and import data that includes master-detail relationships with the SObject Tree Save API. Perform simple CRUD operations on individual records with the REST API.",
+            "subtopics": {
+              "bulk": {
+                "description": "manipulate records using the bulk API"
+              },
+              "record": {
+                "description": "manipulate records using the enterprise API"
+              },
+              "soql": {
+                "description": "fetch records using SOQL"
+              },
+              "tree": {
+                "description": "import/export records using the tree API"
+              }
+            }
+          }
+        }
+      }
+    }
   },
   "files": [
     "/lib",
     "/messages",
+    "/schema",
     "/oclif.manifest.json"
   ],
-  "homepage": "https://github.com/salesforcecli/data",
-  "keywords": [
-    "force",
-    "salesforce",
-    "sfdx",
-    "salesforcedx",
-    "sfdx-plugin"
-  ],
-  "license": "BSD-3-Clause",
-  "repository": "salesforcecli/data",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/salesforcecli/plugin-data.git"
+  },
+  "bugs": "https://github.com/forcedotcom/cli/issues",
+  "publishConfig": {
+    "access": "public"
+  },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "build": "lerna run build",
-    "ci-docs": "yarn sf-ci-docs",
-    "clean": "lerna run clean",
-    "clean-all": "lerna run clean-all",
-    "compile": "lerna run compile",
-    "docs": "lerna run docs",
-    "format": "lerna run format",
-    "install": "sf-lerna-install",
-    "lint": "lerna run lint",
-    "lint-fix": "lerna run lint -- --fix",
-    "postclean-all": "yarn sf-clean all",
-    "postinstall": "yarn bootstrap",
-    "prepack": "lerna run prepack",
-    "test": "lerna run test",
-    "test:nuts": "lerna run test:nuts"
+    "build": "sf-build",
+    "clean": "sf-clean",
+    "clean-all": "sf-clean all",
+    "clean:lib": "shx rm -rf lib && shx rm -rf coverage && shx rm -rf .nyc_output && shx rm -f oclif.manifest.json",
+    "compile": "sf-compile",
+    "docs": "sf-docs",
+    "format": "sf-format",
+    "lint": "sf-lint",
+    "postpack": "shx rm -f oclif.manifest.json",
+    "posttest": "yarn lint && yarn test:deprecation-policy && yarn test:command-reference",
+    "prepack": "sf-build",
```

Comment:
Should this be changed to `sf-prepack`? Like in the template [L94](https://github.com/salesforcecli/plugin-template/blob/main/package.json#L94)

Reply:
Also, should i add `"prepare": "sf-install",`? [L95](https://github.com/salesforcecli/plugin-template/blob/main/package.json#L95)

Reply:
When in doubt about the template, I refer to something running in the real world like https://github.com/salesforcecli/plugin-user

[feel free to PR the template as well to keep it correct.  It's often lagging]